Makefile

```diff
@@ -7,3 +10,11 @@ protobuf:
 	cd Tests/Protobuf; \
 	protoc test_scenarios.proto --swift_out=../CombineGRPCTests/Generated/; \
 	protoc test_scenarios.proto --grpc-swift_out=../CombineGRPCTests/Generated/
+
+protobuf_docker:
+	mkdir -p Tests/CombineGRPCTests/Generated
+
+	docker run --rm -v "${PWD}":/protobuf -w="/protobuf" cyborgthefirst/grpc-swift:1.0.0-alpha.9 \
+	protoc ${PROTO_DIR}/*.proto --swift_opt=FileNaming=DropPath,Visibility=Public \
+	--swift_out=${OUT_DIR} \
+	--grpc-swift_out=FileNaming=DropPath,Visibility=Public,Server=true,Client=true:${OUT_DIR}
```

Comment:
Should we omit explicitly listing these out if they are the default options?

```
FileNaming=DropPath,Visibility=Public,Server=true,Client=true
```

Reply:
Yes definitely! Did not realise that I added the default parameters when testing.